### PR TITLE
Test Python 3.6

### PR DESCRIPTION
**Pull request by ocefpaf**
Upgrade the default environment to `Python 3.6`

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -30,11 +30,6 @@ install:
   - conda env create --quiet --file "%ENV%"
   - activate "%ENV_NAME%"
   - conda install console_shortcut
-  # Fortran cells.
```
Comment (ocefpaf) [inline appveyor.yml]: We don't have any `Fortran` notebook yet and this does not work with latest Python. So I am removing this and in the future we can experiment with `msys2` instead.

**Comment by ocefpaf** [comment]
The failure on Windows seems to be a time out as sometimes passes. So merging this...

**Comment by lukecampbell** [comment]
Is there such a thing as a fortran notebook?

**Comment by ocefpaf** [comment]
> Is there such a thing as a fortran notebook?

You can use both the Fortran magic in cells as well a language kernel.
See this example: https://github.com/ZedThree/jupyter-fortran-kernel/blob/master/example-notebook.ipynb

**Comment by lukecampbell** [comment]
whoa

**Pull request merged**